### PR TITLE
Add Group Set Custom Fields

### PR DESCRIPTION
**Pull request by M-R-Epstein**
Needed to add this real quick for a project I'm working on and figured I'd make a PR.

### CODE REVIEW

**Comment by jadolg** [comment]
Hey, @M-R-Epstein thanks for your contribution :)
Looks like some of the checks have failed (Mostly about styling) 
If you could take a look and fix it 
https://travis-ci.org/github/jadolg/rocketchat_API/jobs/712629680
and
https://deepsource.io/gh/jadolg/rocketchat_API/run/d84b9463-0537-4b27-9613-0db67e394b7f/python/
Also, writing a test for your code will be great as it assures its maintainability ;)

**Comment by M-R-Epstein** [comment]
Sorry about the extra commits. I didn't set up a whole environment for this and editing code through the github interface is not great.

File: tests/test_groups.py [hunk 1]
```diff
@@ -216,6 +216,27 @@ def test_groups_set_type(logged_rocket):
...
+        'Custom fields do not match'
+
+    groups_set_custom_fields = logged_rocket.groups_set_custom_fields(
+        custom_fields, room_name=test_group_name).json()
+    assert groups_set_custom_fields.get('success')
+    assert groups_set_custom_fields.get('group').get('customFields') == custom_fields, \
+        'Custom fields do not match'
+
+    with pytest.raises(RocketMissingParamException):
+        logged_rocket.logged_rocket.groups_set_custom_fields(custom_fields)
```
Comment (jadolg) [inline tests/test_groups.py:237]: There's a repeated `logged_rocket.` here which is making the tests fail.
For some weird reason, it's not showing in the travis check but it is reflected in the coverage analysis.
Once this is corrected I'll merge your changes :)

Comment (M-R-Epstein) [reply tests/test_groups.py]: Durf, sorry about that.

Comment (jadolg) [reply tests/test_groups.py]: no prob ;)

**Comment by jadolg** [comment]
LGTM

**Pull request merged**